### ISSUE TITLE
Reveal "alpha" slider

### ISSUE THREAD

**Issue by sharktacos** [vidarn/ThunderLoom#14]
I've been testing the "alpha" slider that Peter revealed for me in a branch of the shader, and can say that it is really helpful for getting "metallic" fabrics like silk  to look right (honestly I was not able to get silk to look right without it). So I'd put in my vote for keeping this exposed in the shader. For example, changing the alpha from the default value of 0.05 to 1 makes the spec basically uniform like a metal, rather than a dielectric material. Thus giving the "metallic" look of a silk.

I would suggest giving it a more descriptive name than "alpha" which indicates its purpose, like the other attribute names in the shader do. Perhaps you could call it "uniform specularity" or maybe even "metalness."

**Comment by sharktacos** [comment]
Here's an image to illustrate, hope it is helpful:
![image](https://user-images.githubusercontent.com/23034637/30941637-4893f8d0-a39b-11e7-9b62-20accb250183.png)

**Top left** is alpha=1.0, colored spec, spec amount 1.0 (so no diffuse) which already looks like silk. This is the basic formula for a metal material. 
**Top right** adds in a second spec (alpha 0.05) for additional detail.
**Bottom left** is the default value of alpha=0.05 which is basically dielectric Fresnel and thus looks too dark with colored spec.
**Bottom right** is with diffuse color added to the dielectric spec. Looks more like plastic than silk.

TL;DR: As the image shows, the alpha slider is really helpful with shiny fabrics like silk.

**Comment by petermcevoy** [comment]
Is now exposed as of commit 4206da6d9ff7f94a8fc8c545ef52cc36622265cd.
Thanks! 

Will try to change the ui in Maya later so it says uniform specularity, later. Thanks for the suggestion.
